test/utils/image/manifest.go

```diff
@@ -214,7 +214,7 @@ func initImageConfigs() map[int]Config {
 	configs[CudaVectorAdd2] = Config{e2eRegistry, "cuda-vector-add", "2.0"}
 	configs[DebianIptables] = Config{buildImageRegistry, "debian-iptables", "v12.1.0"}
 	configs[EchoServer] = Config{e2eRegistry, "echoserver", "2.2"}
-	configs[Etcd] = Config{gcRegistry, "etcd", "3.4.7"}
+	configs[Etcd] = Config{gcRegistry, "etcd", "3.4.9"}
```

Comment:
does this image exist?

Reply:
this is pointing at `k8s.gcr.io`... does the 3.4.9 image exist there?

Reply:
```suggestion
	configs[Etcd] = Config{prodRegistry, "etcd", "3.4.9"}
```